Build doesn't work with MSBuild 16 / VS 2019

When trying to build the project with the MSBuild from VS2019, I'm getting the following messages:

```console
$ msbuild /p:Configuration=Release
C:\Program Files\dotnet\sdk\2.2.300\Sdks\Microsoft.NET.Sdk\Sdk\Sdk.targets(41,3): error MSB4019: The imported project "C:\Program Files (x 86)\Microsoft Visual Studio\2019\Community\MSBuild\16.0\Bin\Microsoft.CSharp.targets" was not found. Confirm that the path in the <Import>  declaration is correct, and that the file exists on disk.
[…]
Build FAILED.
```

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -1,3 +1,5 @@
-﻿image: Visual Studio 2017
+﻿image:
+    - Visual Studio 2017
+    - Visual Studio 2019
 before_build:
-- nuget restore
+    - nuget restore

--- a/src/WpfMath.Example/WpfMath.Example.csproj
+++ b/src/WpfMath.Example/WpfMath.Example.csproj
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Sunburst.NET.Sdk.WPF/1.0.47">
+﻿<Project Sdk="Sunburst.NET.Sdk.WPF.Patched/1.0.49">
     <PropertyGroup>
         <OutputType>WinExe</OutputType>
         <TargetFramework>net40</TargetFramework>

--- a/src/WpfMath/WpfMath.csproj
+++ b/src/WpfMath/WpfMath.csproj
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Sunburst.NET.Sdk.WPF/1.0.47">
+﻿<Project Sdk="Sunburst.NET.Sdk.WPF.Patched/1.0.49">
     <PropertyGroup>
         <TargetFramework>net40</TargetFramework>
         <LangVersion>7.2</LangVersion>

